src/niswitch/metadata/config.py

```diff
@@ -18,6 +18,8 @@
     },
     'context_manager_name': {
         'task': 'scan',
+        'initiate': 'initiate_scan',
```

Comment:
This refers to a function using its python name.
In all other places we use the C names, for example init_function (see line 24) or when specifying parameters for buffer retrieval mechanisms.
We should be consistent about this and it seems like C names as found in metadata is the way to go.